### DISCUSSION TITLE
为什么安装之后微信打不开了？

### DISCUSSION THREAD

**Discussion by sbywd** [MustangYM/WeChatExtension-ForMac#1855] [Q&A]
<img width="260" alt="iShot_2024-02-09_22 58 54" src="https://github.com/MustangYM/WeChatExtension-ForMac/assets/121930373/c11b0990-c2a9-4ae6-ab58-e16fb6faa2e8">
系统是macOS14.4

**Comment by ETET114514** [comment]
codesign -f -s - --deep /Applications/WeChat.app
